Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -1530,12 +1528,6 @@
       </AppxPackagePayload>
     </ItemGroup>
   </Target>
-  <!-- https://weblogs.asp.net/rweigelt/disable-warnings-in-generated-c-files-of-uwp-app -->
```

Comment:
@Nirmal4G just wanted to check here that this issue has been resolved since we originally put the app together? Was that why you removed it?

Reply:
I removed it to test whether it builds successfully. It does but just to be safe, I have moved this logic to a separate targets file in #3894.